ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -758,9 +783,21 @@ func resourceIBMContainerVpcClusterUpdate(d *schema.ResourceData, meta interface
 			return err
 		}
 
-		if err := ClusterClient.VPCs().SetOutboundTrafficProtection(clusterID, outbound_traffic_protection, Env); err != nil {
-			return err
+		if d.HasChange(DisableOutboundTrafficProtectionFlag) {
+			outbound_traffic_protection := !d.Get(DisableOutboundTrafficProtectionFlag).(bool)
```

Comment:
why we are storing opposite value of DisableOutboundTrafficProtectionFlag

Reply:
It's a legacy stuff, I didn't change the functionality here. On the other hand if I'm right the `DisableOutboundTrafficProtectionFlag` was introduced first, and after that in an upcoming change the opposite value of this was needed. It's a bit wierd, but this is how it's working.